pkgs/development/libraries/pipewire/default.nix

```diff
@@ -5,6 +5,7 @@
 , meson
 , ninja
 , systemd
+, enableSystemd ? true
```

Comment:
Can you add a note here, just something like "used by Spectrum" or whatever, so it doesn't get yeeted in the next cleanup?

Reply:
I've added a comment saying I volunteer to maintain it.